runtime/server/downloads.go

```diff
@@ -76,7 +76,12 @@ func (s *Server) Export(ctx context.Context, req *runtimev1.ExportRequest) (*run
 		req.BakedQuery = ""
 	}
 
-	tkn, err := s.generateDownloadToken(req, auth.GetClaims(ctx).SecurityClaims())
+	tkn, err := s.generateDownloadToken(&runtimev1.DownloadReportRequest{
+		InstanceId: req.InstanceId,
+		Limit:      req.Limit,
+		Format:     req.Format,
+		Query:      req.Query,
+	}, auth.GetClaims(ctx).SecurityClaims())
```

Comment:
Feels unintuitive to use a report-specific type for a non-report export.

Maybe the `generateDownloadToken` will need a way of distinguishing between token types? Ideally it could just receive a copy of `ExportReportRequest` then, so we don't need this separate `DownloadReportRequest` type (which from a naming perspective is quite confusing)

Reply:
Yeah may be the name of the proto message is not the best that may be changed. However to make this generic, we will have to support different token types ? We can also simply add the `ExportReportRequest` but then report spec will need to be fetched in the `downloadHandler` to resolve the actual query.
```
message ExportReportRequest {
  string instance_id = 1;
  string report = 2;
  int64 limit = 3;
  ExportFormat format = 4;
  google.protobuf.Timestamp execution_time = 5;
}

```